### DISCUSSION TITLE
Learning tools we love

### DISCUSSION THREAD

**Discussion by avidrucker** [knowledge-for-good/knowledgeforgood#11] [General]
I have encountered a number of learning tools throughout the years. I will share some with you down below, and I invite you to share what learning tools you have found helpful as well 😃

**Comment by avidrucker** [comment]
Anki (a spaced repetition system) https://apps.ankiweb.net/

**Comment by avidrucker** [comment]
Khan Academy (an online learning platform for math and other things) https://www.khanacademy.org/

**Comment by avidrucker** [comment]
Renshuu (an online learning platform for learning Japanese) https://www.renshuu.org/

**Reply by seekanddefine** [reply]
I like this because it is cute. Earlier, I was trying to say that I hope whatever we build has an element of fun in a way that everyone can engage with it (ex. Steven Universe), but I think the Asian, minimalist, anime vibe is a great way to go.
